README.md

```diff
@@ -7,6 +7,14 @@ Similar to [torchdiffeq](https://github.com/rtqichen/torchdiffeq), algorithms in
   <img width="600" height="450" src="./assets/latent_sde.gif">
 </p>
 
+## Requirements
+- torch>=1.6.0
+- blist
+- numpy>=1.19.1
+- boltons>=20.2.1
+
+These requirements are specified in `setup.py` using `install_requires`.
+
```

Comment:
Usually I wouldn't state requirements in the readme unless some action is necessary on the user's part. (Additionally this means we have two identical lists to maintain going forward.)

Reply:
I don't see this as a big problem. I'm happy to put in the work to maintain this if you find it disturbing.

Reply:
More a neatness thing than anything really. Can we maybe put them lower down? Basically, they're the first thing people see when opening the README, and thus implicitly the most important part of the README, which seems wrong.

I'd suggest promoting the GBM example (of a couple sections down) to this spot.

Reply:
I still don't see it as a problem to include these at the beginning of the README page. It shouldn't hurt to list requires in companion with installation guide.

Reply:
Okey dokey then.